app/src/main/res/layout/edit_recipe_bottom_sheet_fragment.xml

```diff
@@ -255,7 +255,7 @@
                 android:hint="Length (optional)"
                 app:endIconMode="clear_text"
                 app:expandedHintEnabled="false"
-                app:helperText="Length, in bytes (16-999)"
+                app:helperText="Length, in bytes (8-999)"
```

Comment:
For passwords, it's 8 characters.  For secrets, the minimum is 16 bytes.  So, I don't think this change is necessary since the type is secret.

Reply:
It's line 232 you needed to change

Reply:
Yeap, i easily mix between the two.